### PR TITLE
contrib/cray: Add IMB tests to libfabric CI

### PR DESCRIPTION
**Pull request by lradeke**
Added Intel MPI benchmarks to run with data validation enabled across libfabric, in addition to providing a wider suite of tests than OMB.

- Modified Jenkinsfile.verbs file
- Added batsgenerator.sh script for quicker and easier .bats file generation
- Added benchmark.template to be used in conjunction with batsgenerator.sh
- Added subsequently generated imb.bats file with all the new Intel MPI benchmarks

@jswaro @swelch

Signed-off-by: Leena Radeke <leena.radeke@hpe.com>

### CODE REVIEW

**Comment by swelch** [comment]
@lradeke - It looks like the OSU XRC tests are failing.

File: contrib/cray/Jenkinsfile.verbs [hunk 1]
```diff
@@ -136,6 +136,7 @@ pipeline {
                 SFT_TEST_RESULTS_EXPECTED = 'expected_'
                 SFT_TEST_RESULTS_PREFIX = 'BUILD_'
                 SFT_TEST_RESULTS_CI = 'sft_ci_results.yaml'
+		FI_VERBS_MIN_RNR_TIMER= '4'
```
Comment (jswaro) [inline contrib/cray/Jenkinsfile.verbs:139]: The indent looks incorrect here.

File: contrib/cray/Jenkinsfile.verbs [hunk 2]
```diff
@@ -468,6 +479,7 @@ pipeline {
         FABTEST_PATH = "${WORKSPACE + '/installs/fabtests'}"
         LIBFABRIC_BUILD_PATH = "${ROOT_BUILD_PATH + '/libfabric'}"
         OMB_BUILD_PATH = "${ROOT_BUILD_PATH + '/osu-micro-benchmarks/stable/libexec/osu-micro-benchmarks/mpi'}"
+	IMB_BUILD_PATH = "${ROOT_BUILD_PATH + '/imb/v2019.6'}"
```
Comment (jswaro) [inline contrib/cray/Jenkinsfile.verbs:482]: Indent here too

File: contrib/cray/Jenkinsfile.verbs [hunk 3]
```diff
@@ -316,19 +317,29 @@ pipeline {
...
                     steps {
-                        tee ('mpi.tap') {
+                        tee ('omb.tap') {
                             timeout(time: 10, unit: 'MINUTES') {
-                                sh '${BATS_INSTALL_PATH}/bats -t contrib/cray/bats/mpi.bats'
+                                sh '${BATS_INSTALL_PATH}/bats -t contrib/cray/bats/omb.bats'
                             }
                         }
+		        tee ('imb.tap') {
+			    timeout(time: 10, unit: 'MINUTES') {
```
Comment (jswaro) [inline contrib/cray/Jenkinsfile.verbs:326]: spacing here too

**Comment by jswaro** [comment]
Check why CI failed first, but it looks fine otherwise.

**Pull request merged**